docs/faq.md

```diff
@@ -87,6 +87,7 @@ If Ollama is run as a systemd service, environment variables should be set using
     ```ini
     [Service]
     Environment="OLLAMA_HOST=0.0.0.0:11434"
+    Environment="OLLAMA_ORIGINS=chrome-extension://*,moz-extension://*,safari-web-extension://*"
```

Comment:
Let's leave this out of this section, since its just general instructions on how to set environment variables on Linux

Reply:
No problem. I would set this configuration every time, but I understand. I've removed it.